staff/henry-novoa/cv-styled/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Curriculum vitae</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <script src="main.js"></script>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="box info">
+      
+      <h2 class="personalHeader">Personal Data</h2>
```

Comment:
as a good practice the classes in css are named separated by (-) like : personal-header

Reply:
I'm on it